templates/server/web/app_test.go

```diff
@@ -0,0 +1,89 @@
+package web
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"project-name/model"
+	"testing"
+)
+
+var expectedTechnologies = []*model.Technology{
+	{
+		Name:    "Tech1",
+		Details: "Details1",
+	},
+	{
+		Name:    "Tech2",
+		Details: "Details2",
+	},
+}
+
+type MockDb struct {
+	called     int
+	returnTech []*model.Technology
```

Comment:
I'd suggest renaming the properties to simply `tech` and `err`.

Reply:
Done